functional_tests/scylla_operator/libs/helpers.py

```diff
@@ -41,6 +41,14 @@ class PodStatuses(Enum):
     CRASH_LOOP_BACK_OFF = 'CrashLoopBackOff'
 
 
+def create_namespace(db_cluster: ScyllaPodCluster, namespace: str) -> None:
+    log.info("Create '%s' namespace", namespace)
+    namespaces = yaml.safe_load(db_cluster.k8s_cluster.kubectl("get namespaces -o yaml").stdout)
+    if not [ns["metadata"]["name"]
+            for ns in namespaces["items"] if ns["metadata"]["name"] == namespace]:
+        db_cluster.k8s_cluster.kubectl(f"create namespace {namespace}")
```

Comment:
I think for debugging purpose, would be good to add warning log that namespace exists and skipping creation.

Reply:
Ok. Will add